<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonoGui</class>
 <widget class="QWidget" name="MonoGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="mainLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="14" column="0" colspan="6">
         <widget class="QMotorStack" name="motors" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>ΔZ (</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="lockDZ">
             <property name="toolTip">
              <string>Lock Z tweak on Z separation change</string>
             </property>
             <property name="text">
              <string>lock)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="13" column="4">
         <widget class="QPushButton" name="moveOut">
          <property name="text">
           <string>OUT</string>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QPushButton" name="moveIn">
          <property name="text">
           <string>IN</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>ΔX</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lockX">
            <property name="toolTip">
             <string>Keep value on energy change</string>
            </property>
            <property name="text">
             <string>lock</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="5">
         <widget class="QLabel" name="readB1B">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QLabel" name="readB1F">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="6">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="readDX">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="Tuner" name="bend2front">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Energy</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Bend back</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="energy">
          <property name="toolTip">
           <string>Photon energy</string>
          </property>
          <property name="styleSheet">
           <string notr="true">font: bold;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="suffix">
           <string>keV</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>17.000000000000000</double>
          </property>
          <property name="maximum">
           <double>195.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Tune</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="Tuner" name="bend2back">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="readTilt2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="readB2B">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Diffraction</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>second</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="5">
         <widget class="QLabel" name="readTilt1">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="6">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="4" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>first</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="readBragg">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="Tuner" name="tuneX">
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_17">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Bend front</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="readB2F">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Δθ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lockBragg">
            <property name="toolTip">
             <string>Keep value on energy change</string>
            </property>
            <property name="text">
             <string>lock</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="9" column="1">
         <widget class="Tuner" name="tilt2">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <double>-3.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Tilt</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="si111">
            <property name="toolTip">
             <string>For 17-100 kEv</string>
            </property>
            <property name="text">
             <string>&lt;111&gt;</string>
            </property>
            <attribute name="buttonGroup">
             <string>diffractions</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="si311">
            <property name="toolTip">
             <string>For 31-195 kEv</string>
            </property>
            <property name="text">
             <string>&lt;311&gt;</string>
            </property>
            <attribute name="buttonGroup">
             <string>diffractions</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="4">
         <widget class="Tuner" name="bend1front">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="12" column="4">
         <widget class="Tuner" name="tuneZ">
          <property name="minimum">
           <double>-5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="6">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="Tuner" name="bend1back">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="Tuner" name="tuneBragg">
          <property name="suffix">
           <string>μRad</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-10000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="Tuner" name="tilt1">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <double>-6.000000000000000</double>
          </property>
          <property name="maximum">
           <double>6.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="3" rowspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="6">
         <widget class="QPushButton" name="advanced_pb">
          <property name="text">
           <string>Show advanced control</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="2">
         <widget class="QLabel" name="label_14">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Position 1st crystal in beam:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Z separation</string>
          </property>
         </widget>
        </item>
        <item row="12" column="5">
         <widget class="QLabel" name="readDZ">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="13" column="5">
         <widget class="QLabel" name="currentInOut">
          <property name="text">
           <string>disconnected</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QMDoubleSpinBox" name="zSeparation">
          <property name="minimum">
           <double>15.000000000000000</double>
          </property>
          <property name="maximum">
           <double>35.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QWidget" name="scheme" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/mono-scheme.png);</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="5">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="readBragg1">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="readBragg2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="readX">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="bigDBragg">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(128, 0, 2);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>meaninglessly large value</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="bigDX">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(128, 0, 2);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>meaninglessly large value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stop">
     <property name="toolTip">
      <string>Stop any motion.</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tuner</class>
   <extends>QDoubleSpinBox</extends>
   <header>tuner.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QMotorStack</class>
   <extends>QWidget</extends>
   <header location="global">qmotorstack.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QMDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">qtpvwidgets.h</header>
   <slots>
    <signal>valueEdited(double)</signal>
    <signal>escaped()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="monoGui.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="diffractions"/>
 </buttongroups>
</ui>
